Algebroids/gap/CategoryFromNerveData.gi

```diff
@@ -665,7 +665,7 @@ InstallMethod( \.,
 end );
 
 ##
-InstallMethod( SetOfObjects,
+InstallMethodForCompilerForCAP( SetOfObjects,
```

Comment:
Does this actually affect the compiled code?

If yes: Attributes of categories are usually resolved by actually computing their value and storing the result in a global variable. So this change creates a race condition between resolving methods installed via `InstallMethodForCompilerForCAP` as functions and resolving attributes of categories as global variables. In this case, we should create a CAP issue to track this and avoid the ambiguity.

If no: Please revert this change to avoid future ambiguity.

Reply:
Yes, it does affect compiled code.